### PR TITLE
Add Explore Metrics button to uploadPackages tab

### LINKED ISSUE #653: 'Explore Metrics' button from `Upload Packages` tab
Add an "Explore Metrics" button in the "Summary of Uploaded Pkgs" after upload, similar to the buttons seen on the database view tab:

image

### PR DESCRIPTION
**Pull request by Robert-Krajcik**
closes #653

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 * Started showing `metric_score`s for each assessment's card, using an html meter (#634)
 * Removed some unneeded/unused dependencies
 * Allow user to create decision automation rules for individual metrics (#483)
+* Add Explore Metrics button from Upload Packages tab (#653)
```
Comment (AARON-CLARK) [inline NEWS.md:14]:
```suggestion
* Add "Explore Metrics" button to Uploaded Pkgs Summary tab on Upload Packages tab (#653)
```

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]
Hi @Robert-Krajcik, this PR is close. I noticed that when I upload a packages initially, the button works. However, if I upload another package afterwards, the button doesn't work anymore. Can this be resolved?

**Comment by Robert-Krajcik** [comment]
Hi @AARON-CLARK 
`I noticed that when I upload a packages initially, the button works. However, if I upload another package afterwards, the button doesn't work anymore. `
I am not able to replicate this.  Seems to work for me.  Maybe you should share your screen with me later today?

**Comment by Robert-Krajcik** [comment]
Hi @AARON-CLARK 
This is working now for me.

**Review by AARON-CLARK — APPROVED** [review]
👍🏼 👍🏼 👍🏼

**Pull request merged**